js/components/interface/query/queryBuilder.js

```diff
@@ -1055,7 +1096,7 @@ define(function (require) {
                               <div className="clearer"></div>
                               <Griddle
                                 showFilter={true}
-                                initialSort={this.sorterColumn}
+                                initialSort={this.sorterColumn.column}
```

Comment:
@ddelpiano same here

Reply:
same reply apply to this, in this case I am passing to griddle the single column.